src/pages/CirclesPage/CirclesPage.tsx

```diff
@@ -152,7 +152,7 @@ const GetStarted = () => {
           </Button>
           <Button
             as={NavLink}
-            to={EXTERNAL_URL_GET_STARTED}
+            to={`//${EXTERNAL_URL_GET_STARTED}`}
```

Comment:
These slashes are weird

Reply:
@CryptoGraffe they were previously getting appended to the url.  like `localhost:3000/http://doc.coor....` why?